task2/script.js

```diff
@@ -0,0 +1,17 @@
+const liElement = document.createElement('li');
+
+const anchorElement = document.createElement('a');
+anchorElement.setAttribute('href', '1.html');
```

Comment:
```
anchorElement.setAttribute('href', '1.html');
```
【IMO】

I thought '.property' would be more appropriate than '.setAttribute'.

```
/* .setAttribute */
anchorElement.setAttribute('href', '1.html');

/* .property */
anchorElement.href = '1.html';
```

-  [Mr.moriken's article](https://github.com/kenmori/handsonFrontend/blob/553595c5e3333a76152c0e76ae9c5cc8293b34e6/html/README.md) said,
”Using set/getAttribute may result in unintended behavior.” 

- There is also [an article](https://hacknote.jp/archives/23219/) that Mr.yamamoto put up on the 
Discord, so please read it😆

Reply:
I found that the basics use "property.id = ◯◯", "property.href = ◯◯".
I would like to change it that way.

Thank you for the review.

Reply:
good!